### ISSUE TITLE
recline_view.js assumes that window.parent.ckan is defined

### ISSUE THREAD

**Issue by clementmouchet** [ckan/ckan#2238]
There is an assumption in `recline_view.js` that means views won't load when embedded in a different website.

l.65
``` javascript
      if (window.parent.ckan.views && window.parent.ckan.views.filters) {
```
This is assumes `window.parent.ckan` is defined, but it's not always defined.
Changing this to:

``` javascript
      if (window.parent.ckan && window.parent.ckan.views && window.parent.ckan.views.filters) {
```
fixes the problem.

I'll submit a pull request asap.

**Comment by amercader** [comment]
Good catch, can you send a PR so we can backport it to a patch release?

**Comment by clementmouchet** [comment]
PR: https://github.com/ckan/ckan/pull/2335

Thanks @amercader

**Comment by amercader** [comment]
Fixed in #2423
